client/src/components/MethodologyFormula/MethodologyFormula.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {useIntl} from 'gatsby-plugin-intl';
+
+import * as METHODOLOGY_COPY from '../../data/copy/methodology';
+import * as styles from './MethodologyFormula.module.scss';
+
+const MethodologyFormula = () => {
+  const intl = useIntl();
+
+  return (
+    <section className={styles.formulaContainer}>
+      <p>
+        {intl.formatMessage(METHODOLOGY_COPY.PAGE.FORMULA_INTRO)}
+      </p>
+
+      <p>
+        {METHODOLOGY_COPY.FORMULA.IF}
```

Comment:
this is a cool use of IF AND and THEN. I was a little confused above in CategoryCard but when it's paired with FORMULA it is very clear. I wonder if it would be helpful to use FORMULA in the names elsewhere (eg `FORMULA_IF` OR `FORMULA.IF`) so that it's always clear what these words are and less likely that they might be confused with their reserve word counterparts (eg `if`, `else`, `then`)

Reply:
Added some comments to make this clearer